shell/browser/ui/file_dialog_gtk.cc

```diff
@@ -101,14 +202,23 @@ class FileChooserDialog {
     if (!settings.filters.empty())
       AddFilters(settings.filters);
 
+    // GtkFileChooserNative does not support preview widgets through the
+    // org.freedesktop.portal.FileChooser portal. In the case of running through
+    // the org.freedesktop.portal.FileChooser portal, anything having to do with
+    // the update-preview signal or the preview widget will just be ignored.
```

Comment:
Is there a particular reason this even bothers to create the image widgets? Seems a bit like a bit of a waste, since it's not doing anything here...

Reply:
@refi64 it was left in because we could be linking against a version of GTK that doesn't support GtkFileChooserNative.

Reply:
Ah apologies, I mis-stated. What I meant to say was that it felt weird to still create the image widgets themselves *when a native dialog is being used*, even though it's ignored we still end up with a useless widget in memory.

Reply:
I'll take a look at the logic and rework it. Thanks for your input.